collection-scripts/gather_core_dumps

```diff
@@ -4,10 +4,31 @@ CORE_DUMP_PATH=${OUT:-"${BASE_COLLECTION_PATH}/node_core_dumps"}
 
 mkdir -p ${CORE_DUMP_PATH}/
 
+function get_dump_off_node { 
+    #Start Debug pod force it to stay up for 60 seconds, supress Stdout 
+    oc debug node/$1 -- /bin/bash -c 'sleep 60' > /dev/null 2>&1 &
+
+    # Make sure pod is avaliable exit after 5 seconds if not 
+    while [[ -z "${DEBUG}"  && $n -le 10 ]]
```

Comment:
you probably want to set n to 0 before this, and unset debug, or you make them both local vars to this function. Otherwise calling this function multiple times may have n already incremented or DEBUG already have a value.

Reply:
Ahh you're right, nice catch, I will convert `debug` and the counter to local variables, and then since each function call is being run in the background, i.e separate processes, there shouldn't be any chance of variable overwriting